src/Snowflake/Connection.php

```diff
@@ -46,6 +46,7 @@ public function __construct(array $options)
 
         $port = isset($options['port']) ? (int)$options['port'] : 443;
         $tracing = isset($options['tracing']) ? (int)$options['tracing'] : 0;
+        $maxBackoffAttempts = isset($options['maxBackoffAttempts']) ? (int)$options['maxBackoffAttempts'] : 5;
```

Comment:
there should be also space after type cast. Unfortunately it is not catched by code style checker.